peekingduck/pipeline/nodes/dabble/utils/load_tracker.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Load Tracker for inference
+"""
+
+from typing import Any, Dict, List
+import logging
+from .tracking_files.iou_tracking import IOUTracking
+from .tracking_files.opencv_tracking import OpenCVTracker
+
+
+class LoadTracker:  # pylint: disable=too-few-public-methods
+    """Loads chosen tracker node."""
+    def __init__(self, tracking_type: str) -> None:
+        super().__init__()
+        self.logger = logging.getLogger(__name__)
+        if tracking_type == "mosse":
```

Comment:
Its cleaner to put in a dictionary say {'iou': IOUTracker, 'mouse': MOSSETracker} and pull out. This way if eventually the number of tracker models increase we can just add to the registry.

Reply:
This is a good idea, I will implement this